src/store/user/authentication.ts

```diff
@@ -0,0 +1,18 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {APP_AUTHENTICATION} from 'constants/redux';
+import {Dict} from 'types/store';
+
+const authentication = createSlice({
+  initialState: {} as Dict<any>,
+  name: APP_AUTHENTICATION,
+  reducers: {
+    setAuthentication: (state, {payload}: PayloadAction<any>) => {
+      const {accessToken} = payload;
+      state.accessToken = accessToken;
```

Comment:
I think we should rename this to `token` or `authToken` as we are already using `accessToken` in backend for some other purpose(GitHub). Not a problem, but just to be clear.

Reply:
I just put `accessToken` for now because I am not sure what will we receive from the API once it's done. So basically it's just a placeholder for now, we'll change it when we implement authentication.